src/test/kotlin/no/nav/syfo/api/AuthenticateSpek.kt

```diff
@@ -0,0 +1,113 @@
+package no.nav.syfo.api
+
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.ContentNegotiation
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpMethod
+import io.ktor.http.HttpStatusCode
+import io.ktor.jackson.jackson
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import io.ktor.server.testing.TestApplicationEngine
+import io.ktor.server.testing.handleRequest
+import io.ktor.util.KtorExperimentalAPI
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.VaultSecrets
+import no.nav.syfo.initRouting
+import no.nav.syfo.testutil.TestDB
+import org.amshove.kluent.shouldEqual
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.util.Base64
+import java.util.concurrent.TimeUnit
+
+@KtorExperimentalAPI
+object AuthenticateSpek : Spek({
+
+    val database = TestDB()
+    val fakeApi = embeddedServer(Netty, 9000) {
```

Comment:
Det kan være fordel å finne en tilfeldig åpen port her, siden 9000 kan være i bruk hvor den bygges og det kan være vanskelig å feilsøke:

val randomPort = ServerSocket(0).use { it.localPort }